net/domain.c

```diff
@@ -46,7 +46,7 @@ int m0_net_domain_init(struct m0_net_domain *dom, struct m0_net_xprt *xprt)
 	m0_list_init(&dom->nd_tms);
 
 	dom->nd_xprt_private = NULL;
-	dom->nd_xprt = xprt;
+	dom->nd_xprt = (struct m0_net_xprt *) xprt;
```

Comment:
I think we should change "struct m0_net_domain::nd_xprt" to const.

Reply:
incorporated